docs/migrating-from-0.2-to-1.0.md

```diff
@@ -372,3 +374,59 @@ The `embedded-hal` project now spans several crates, where some functionality ha
 | [embedded-io-async](./embedded-io-async)       | [![crates.io](https://img.shields.io/crates/v/embedded-io-async.svg)](https://crates.io/crates/embedded-io-async) | [![Documentation](https://docs.rs/embedded-io-async/badge.svg)](https://docs.rs/embedded-io-async) | I/O traits, async version  |
 | [embedded-io-adapters](./embedded-io-adapters)       | [![crates.io](https://img.shields.io/crates/v/embedded-io-adapters.svg)](https://crates.io/crates/embedded-io-adapters) | [![Documentation](https://docs.rs/embedded-io-adapters/badge.svg)](https://docs.rs/embedded-io-adapters) | Adapters between the [`embedded-io`](https://crates.io/crates/embedded-io) and [`embedded-io-async`](https://crates.io/crates/embedded-io-async) traits and other IO traits (`std`, `tokio`, `futures`...)  |
 
+## Supporting both 0.2 and 1.0 in the same HAL
+
+It is strongly recommended that HAL implementation crates provide implementations for both the `embedded-hal` v0.2 and v1.0 traits.
+This allows users to use drivers using either version seamlessly.
+
+The way you do it is adding a dependency on both versions in `Cargo.toml` like this:
+
+```toml
+[dependencies]
+embedded-hal-02 = { package = "embedded-hal", version = "0.2.7", features = ["unproven"] }
+embedded-hal-1 = { package = "embedded-hal", version = "1.0" }
```

Comment:
why do you want to rename `embedded-hal` to `embedded-hal-v1`? presuming that support for v0 will be dropped in the future v1 should be the only one remaining and then there's no need to refer to it as anything other than `embedded-hal`

Reply:
I did it like this in Embassy because I thought it's more explicit. When the HAL drops 0.2 support, renaming is trivial with a global find-and-replace.